blocks/arrow-icon/arrow-icon.css

```diff
@@ -8,3 +8,11 @@
 .arrow-icon:hover {
   background: url("./yellow-arrow.png") center center no-repeat;
 }
+
+@media only screen and  (max-width: 320px) {
+  .arrow-icon {
+    font-size: 14px;
+    width: 26px;
+    height: 22px;
```

Comment:
The arrow is too big compared to the figma design

Reply:
fixed